Lykke.Service.LykkeService/client/Lykke.Service.LykkeService.Client/AutofacExtension.cs

```diff
@@ -1,34 +1,25 @@
 ﻿using System;
 using Autofac;
-using Autofac.Core;
 using JetBrains.Annotations;
-using Lykke.Common.Log;
+using Lykke.HttpClientGenerator;
 
 namespace Lykke.Service.LykkeService.Client
 {
     [PublicAPI]
     public static class AutofacExtension
```

Comment:
I explicitly didnt add these autofac extensions to the HFT api client since I don't want to force users to use the HttpClientGenerator. Its up to them if they use the HttpClientGenerator or the Refit RestClient functionality. But this might be a bit different for services that are used internally.